homeassistant/components/rflink/sensor.py

```diff
@@ -61,25 +66,110 @@
         state_class=SensorStateClass.MEASUREMENT,
     ),
     SensorEntityDescription(
-        key="average_windspeed",
-        name="Average windspeed",
-        device_class=SensorDeviceClass.WIND_SPEED,
+        key="battery",
+        name="Battery",
+        icon="mdi:battery",
+    ),
+    SensorEntityDescription(
+        key="co2_air_quality",
+        name="CO2 air quality",
+        device_class=SensorDeviceClass.CO2,
```

Comment:
What's the unit of measurement for the CO2 sensor? When using a device class we need to know that the measurement is using an approved unit of measurement.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes

Reply:
I've open https://github.com/home-assistant/core/pull/82729 to add unit of measurement where missing